src/ResourceManager/Profile/Commands.Profile/Environment/AddAzureRMEnvironment.cs

```diff
@@ -136,7 +164,11 @@ public override void ExecuteCmdlet()
             newEnvironment.Endpoints[AzureEnvironment.Endpoint.AzureDataLakeStoreFileSystemEndpointSuffix] = AzureDataLakeStoreFileSystemEndpointSuffix;
             newEnvironment.Endpoints[AzureEnvironment.Endpoint.AdTenant] = AdTenant;
             newEnvironment.Endpoints[AzureEnvironment.Endpoint.GraphEndpointResourceId] = GraphAudience;
-            WriteObject((PSAzureEnvironment)profileClient.AddOrSetEnvironment(newEnvironment));
+
+            if (ShouldProcess(Name, "Adding new environment"))
```

Comment:
SHouldProcess is not a breaking change - default execution is unaffected.  Can we please add the environment settings we are using to the SHoudlProcess string?